src/include/cursor_inline.h

```diff
@@ -565,3 +565,22 @@ __cursor_row_slot_key_return(WT_CURSOR_BTREE *cbt, WT_ROW *rip, WT_CELL_UNPACK_K
     cbt->rip_saved = rip;
     return (0);
 }
+
+/*
+ * We need a tombstone to mark deleted records, and we use the special value below for that purpose.
+ * We use two 0x14 (Device Control 4) bytes to minimize the likelihood of colliding with an
+ * application-chosen encoding byte, if the application uses two leading DC4 byte for some reason,
+ * we'll do a wasted data copy each time a new value is inserted into the object.
+ */
+static const WT_ITEM __tombstone = {"\x14\x14", 2, NULL, 0, 0};
```

Comment:
If this moves to the shared location here it needs a better name. We use tombstones for a number of different purposes across the codebase - this one needs a name that starts with `wt` to indicate it's shared and that reflects more specifically what the tombstone is used for (keeping track of deleted items in multi-layered tables).

Reply:
renamed __wt_tombstone